pkg/runner/runner_factory.go

```diff
@@ -0,0 +1,12 @@
+package runner
```

Comment:
Please add [the license header](https://github.com/LinuxSuRen/api-testing/blob/master/LICENSE) into the every new files. See also https://github.com/LinuxSuRen/api-testing/blob/e2f2d323f03eee2698c0c274252a8880b8222f29/pkg/secret/types.go